tests/integration/test_constraints.py

```diff
@@ -62,6 +64,9 @@ def test_failing_constraints():
         ScalarInequality('j', '>=', 5.5),
         Range('a', 'b', 'c'),
         ScalarRange('a', 0, 0),
+        create_custom_constraint(
+            lambda _, x: pd.Series([True if x_i > 0 else False for x_i in x['k']])
+        )('k')
```

Comment:
minor: can we break this into two lines: One to create the class and one to initialize?